astrogrism/simulate/simulate.py

```diff
@@ -0,0 +1,90 @@
+from os import environ
+from pathlib import Path
+from shutil import copy
+from tarfile import open as tar_open
+from tempfile import gettempdir
+from warnings import warn
+
+from astropy.utils.data import download_file
+from synphot import Observation
+
+
+def generate_simulation_spectrum(grism, detector=None, temp_path=gettempdir()):
```

Comment:
Grammatical nitpick - would this sound better as `generate_simulated_spectrum`? Or `generate_spectrum_for_simulation`? That's getting rather long for a function name though, hm.

Reply:
Ahh, I see how you're reading it. Since I was working on this and #19 at the same time, I read it as "generate the simulation's spectrum". Honestly, maybe just `generate_synthetic_spectrum` might cut to the chase?

Reply:
I think that's perfect.